src/config/kyrgyzstan/layers.json

```diff
@@ -214,18 +214,50 @@
     "server_layer_name": "mxd13a2_vim_dekad",
     "base_url": "https://odc.ovio.org/",
     "date_interval": "days",
+    "wcsConfig": {
+      "scale": 0.0001,
+      "offset": 0,
+      "pixelResolution": 64
+    },
     "opacity": 0.7,
-    "legend_text": "LEGEND",
+    "legend_text": "Normalized Difference Vegetation Index (NDVI) derived from MODIS TERRA/AQUA",
     "legend": [
```

Comment:
Should we add an issue to create "standardized/shared" legends and wcs configs?

Reply:
@ericboucher - yes! https://github.com/WFP-VAM/prism-frontend/issues/358